app/__init__.py

```diff
@@ -4,10 +4,9 @@
 from flask import Flask
 from flask_migrate import Migrate
 from flask_sqlalchemy import SQLAlchemy
-
+from flask_bootstrap import Bootstrap
```

Comment:
import from established module must be in one block

Reply:
But there are all flask modules, aren't they?

Reply:
or you are talking about missing empty line before config import?